frontend/components/views/NewSchemaView/NewSchemaView.tsx

```diff
@@ -70,17 +68,18 @@ const NewSchemaViewContainer = styled.div`
   flex-direction: column;
 `;
 
-// props interface
+// Define the props interface for NewSchemaView component
```

Comment:
NOTE: If we had time we would fix the typing of AppState to properly be extendable